code/modules/mob/living/simple_animal/hostile/syndicate_mobs.dm

```diff
@@ -357,23 +353,23 @@
 	speed = 1.5
 	death_sound = 'sound/mecha/mechmove03.ogg'
 	loot = list(/obj/effect/mob_spawn/human/corpse/syndicatecommando,
-				/obj/item/salvage/loot/syndicate,
 				/obj/effect/decal/cleanable/blood/innards,
 				/obj/effect/decal/cleanable/blood,
 				/obj/effect/gibspawner/generic,
 				/obj/effect/gibspawner/generic)
+	loot_override = list(/obj/effect/mob_spawn/human/corpse/syndicatecommando,
+            		/obj/item/salvage/loot/syndicate,
+                	/obj/effect/decal/cleanable/blood/innards,
+                	/obj/effect/decal/cleanable/blood,
+                	/obj/effect/gibspawner/generic,
+                	/obj/effect/gibspawner/generic)
+
 
 /mob/living/simple_animal/hostile/syndicate/melee/space/Process_Spacemove(movement_dir = 0)
 	return TRUE
 
 /mob/living/simple_animal/hostile/syndicate/melee/space/Initialize(mapload)
```

Comment:
Since we're just calling the parent here we don't really need to have this defined in the first place.